### PR TITLE
Add due date to the mock data for day view page

### PR DESCRIPTION
**Pull request by fzc233**
add due date interface and the page can not show the out of date task

### CODE REVIEW

File: blotztask-ui/src/app/task-dayview/models/taskDto.ts [hunk 1]
```diff
@@ -5,4 +5,5 @@ export interface taskDto {
   isDone: boolean;
   createdAt: string;
   updatedAt: string;
+  dueDate: string;
```
[the remark this answers is not in the document]
Comment (fzc233) [reply blotztask-ui/src/app/task-dayview/models/taskDto.ts]: I already change the String data type to the Date type

File: blotztask-ui/src/app/task-dayview/page.tsx [hunk 2]
```diff
@@ -7,6 +7,13 @@ import { z } from 'zod';
 import { taskDto } from './models/taskDto';
 import { TaskDTO, taskDTOSchema } from './schema/schema';
 import { TaskForm } from './components/form';
+import React from 'react';
+
+// 获取当前日期的字符串形式，格式为 YYYY-MM-DD
```
[the remark this answers is not in the document]
Comment (fzc233) [reply blotztask-ui/src/app/task-dayview/page.tsx]: sure

File: blotztask-ui/.env.production [hunk 3]
```diff
@@ -1,2 +1,3 @@
 NEXT_PUBLIC_API_BASE_URL=https://wapp-blotztaskapp.azurewebsites.net
 NEXT_PUBLIC_API_BASE_URL_WITH_API=https://wapp-blotztaskapp.azurewebsites.net/api
```
Comment (Ben0189) [inline blotztask-ui/.env.production:2]: dont need new line here

File: blotztask-ui/.gitignore [hunk 4]
```diff
@@ -34,6 +34,3 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
-
-# vscode
```
Comment (Ben0189) [inline blotztask-ui/.gitignore:38]: why remove ?

File: blotztask-ui/public/assets/images/logo.svg
Comment (Ben0189) [inline blotztask-ui/public/assets/images/logo.svg:1]: why remove?

**Pull request closed without merging**